<ui version="4.0" >
 <class>BoundaryDivisionForm</class>
 <widget class="QDialog" name="BoundaryDivisionForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Divide boundary</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../Mesh3D.qrc" >:/icons/divide.png</iconset>
  </property>
  <widget class="QGroupBox" name="angleGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>Sharp edge</string>
   </property>
   <widget class="QLabel" name="angleLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Angle:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="angleDegreeEdit" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>20</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="degreeLabel" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>deg</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="divideButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Divide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>100</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Mesh3D.qrc" />
 </resources>
 <connections/>
</ui>
